<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configBatFrame</class>
 <widget class="QFrame" name="configBatFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QWidget" name="batWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#batWidget
{
background-image: url(:/images/cfgBat_Bg.png);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="batEditBut">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>555</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#batEditBut
{
background-image: url(:/images/cfgEditBut_Bt.png);
}
#batEditBut:pressed
{
background-image: url(:/images/cfgEditButDn_Bt.png);
}
#batEditBut:focus{outline: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="batOkBut">
   <property name="geometry">
    <rect>
     <x>737</x>
     <y>555</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#batOkBut
{
background-image: url(:/images/cfgOkBut_Bt.png);
}
#batOkBut:pressed
{
background-image: url(:/images/cfgOkButDn_Bt.png);
}
#batOkBut:focus{outline: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="batCancelBut">
   <property name="geometry">
    <rect>
     <x>915</x>
     <y>555</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#batCancelBut
{
background-image: url(:/images/cfgCancel_Bt.png);
}
#batCancelBut:pressed
{
background-image: url(:/images/cfgCancelDn_Bt.png);
}
#batCancelBut:focus{outline: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
